core/src/integration/java/com/radixdlt/integration/steady_state/simulation/consensus_ledger_epochs/ProposerTimestampInaccurateClockAndLeaderDownTest.java

```diff
@@ -194,7 +194,7 @@ public TimeSupplier timeSupplier() {
                 >= 1);
       } else if (node.equals(downNode)) {
         // The down node shouldn't process any consensus events
-        assertEquals(0, (long) counters.bft().successfullyProcessedVotes().get());
+        assertEquals(0, (long) counters.bft().successfullyProcessedVotes().getSum());
```

Comment:
looks unintentional?

Reply:
yeah, this comes from my other PR (pulled in now)